src/System.Diagnostics.Process/src/PinvokeAnalyzerExceptionList.analyzerdata.uap

```diff
@@ -1,7 +1,6 @@
 advapi32.dll!OpenProcessToken
 kernel32.dll!GetProcessAffinityMask
-kernel32.dll!K32EnumProcessModules
 kernel32.dll!SetProcessAffinityMask
 kernel32.dll!SetThreadAffinityMask
-ntdll.dll!NtQueryInformationProcess
+kernel32.dll!QueryFullProcessImageName
```

Comment:
@danmosemsft Can we use this on UWP? I'm not sure how to find out if this is approved.